src/foundation/Icon/Generated/IconBell12.tsx

```diff
@@ -11,21 +11,12 @@ function IconBell12(props: GeneratedIconProps): React.ReactElement<GeneratedIcon
             name="IconBell12"
             {...props}
         >
-            <g fill="none">
-                <g clipPath="url(#prefix__a)">
-                    <path
-                        fill="currentColor"
```

Comment:
But it looks like the fill on the path was (and will still be) `currentColor`. At least in this case